token/options.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/xmidt-org/themis/key"
 )
 
+const (
+	DefaultTrustLevelNoCertificates   = 0
+	DefaultTrustLevelExpiredUntrusted = 100
+	DefaultTrustLevelExpiredTrusted   = 1000
+	DefaultTrustLevelUntrusted        = 1000
+	DefaultTrustLevelTrusted          = 1000
+)
```

Comment:
```go
const (
	DefaultTrustLevelNoCertificates   = 0
	DefaultTrustLevelExpiredUntrusted = 0
	DefaultTrustLevelExpiredTrusted   = 500
	DefaultTrustLevelUntrusted        = 0
	DefaultTrustLevelTrusted          = 1000
)
```

I think these are the default values we want so we default to only possibly trusting a cert we can prove chains back up to a trusted root CA.